app/controllers/line_items_controller.rb

```diff
@@ -0,0 +1,50 @@
+class LineItemsController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    chosen_product = Product.find(params[:product_id])
+    current_cart = @current_cart
```

Comment:
Nie musisz tego przypisywac do dodatkowej zmiennej raczej, jak usuniesz te linie i zastapisz w reszcie kodu
```ruby
current_cart na @current_cart
```
to powinno banglać i być schludniej